### PR TITLE
[NIL-113] Acceptance tests and content

### PR DESCRIPTION
**Pull request by mchalejeiwc**
This changeset consists of:

1) Acceptance tests for NIL Hub page and new Indicator search tests.
2) Test content and supporting bits/classes
3) Slight rework of the NI Hub links for easier seeding of related links, and some additional IGB landing content.

### CODE REVIEW

File: site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java [hunk 1]
```diff
@@ -226,7 +226,8 @@ private Constraint publicationSystemConstraint(String query) {
     private Constraint indicatorLibraryConstraint(String query) {
         return or(
             constraint("nationalindicatorlibrary:title").contains(query),
-            constraint("nationalindicatorlibrary:methodology").contains(query)
+            constraint("nationalindicatorlibrary:definition").contains(query),
+            constraint("common:SearchableTags").contains(query)
```
Comment (jacsamell) [inline site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java]: Don't need `common:SearchableTags` as it's already been added in the publicationsystem one, really they should just be one list so we don't get duplicates like this

Comment (mchalejeiwc) [reply site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java]: Hmm. I see that - although one list covering multiple doc types (namespaces) seems ugly.

Comment (jacsamell) [reply site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java]: Or split common:... out into it's own list?

Comment (mchalejeiwc) [reply site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java]: I know that the NIL stuff will be extended soon, so I'm going with 3 lists.

Comment (jacsamell) [reply site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java]: 👍

File: site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java [hunk 2]
```diff
@@ -215,8 +217,7 @@ private Constraint publicationSystemConstraint(String query) {
         return or(
             constraint("publicationsystem:Title").contains(query),
             constraint("publicationsystem:Summary").contains(query),
-            constraint("publicationsystem:KeyFacts").contains(query),
-            constraint("common:SearchableTags").contains(query)
+            constraint("publicationsystem:KeyFacts").contains(query)
```
Comment (wojtek-oledzki) [inline site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java]: looks like a merge gremlin, please revert and include `common:SearchableTags`

File: acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/site/SitePage.java [hunk 3]
```diff
@@ -8,6 +8,7 @@
 import uk.nhs.digital.ps.test.acceptance.pages.site.ps.DatasetPageElements;
 import uk.nhs.digital.ps.test.acceptance.pages.site.ps.PublicationPageElements;
 import uk.nhs.digital.ps.test.acceptance.pages.site.ps.SeriesPageElements;
+import uk.nhs.digital.ps.test.acceptance.pages.site.ps.IndicatorPageElements;;
```
Comment (wojtek-oledzki) [inline acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/site/SitePage.java]: double ';;'

File: site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java (hunk 2 above)
Comment (mchalejeiwc) [reply site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java]: No it's pulled in from the 'commonConstraint' function - see earlier conversations about this.

**Pull request merged**